examples/dlint/plugin.rs

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 the Deno authors. All rights reserved. MIT license.
+use deno_lint::linter::Context;
+use deno_lint::rules::LintRule;
+use swc_ecmascript::ast::ImportDecl;
+use swc_ecmascript::visit::Node;
+use swc_ecmascript::visit::Visit;
+
+pub struct WarnRawGitImport;
+
+impl LintRule for WarnRawGitImport {
+  fn new() -> Box<Self> {
+    Box::new(WarnRawGitImport)
+  }
+
+  fn code(&self) -> &'static str {
+    "no-raw-git-import"
+  }
+
+  fn lint_module(
+    &self,
```

Comment:
It's a regular rule, just outside of `deno_lint` lib

Reply:
Yup - the goal is to port than to JS